packages/tocco-ui/src/Select/IndicatorsContainer.js

```diff
@@ -16,11 +16,12 @@ const IndicatorsContainer = props => {
     <components.IndicatorsContainer {...restProps}>
       {children}
       {openAdvancedSearch
+      && !readOnly
```

Comment:
i like the icon (openadvancedsearch) to distinguish remote and select fields. This wouldn't be the possible in case of readonly. The question is, does if have to be?

Reply:
If readonly, only the current selection is relevant. Selected values defines the characteristic, not the none selected items. Listing none accessible actions are from minor interest or disturbing. 

In case that identifying a field's type is a real need than I would suggest to display an icon.